### ISSUE TITLE
Feature Request: Import functionality for aws_waf_ipset

### ISSUE THREAD

**Issue by iancward** [terraform-providers/terraform-provider-aws#4279]
Hi there,

Thank you for opening an issue. Please note that we try to keep the Terraform issue tracker reserved for bug reports and feature requests. For general usage questions, please see: https://www.terraform.io/community.html.

### Terraform Version
Terraform v0.11.7
+ provider.aws v1.15.0

### Affected Resource(s)
- aws_waf_ipset

### Terraform Configuration Files
```hcl
resource "aws_waf_ipset" "ipset" {
  name = "tfIPSet"

  ip_set_descriptors {
    type  = "IPV4"
    value = "192.0.7.0/24"
  }

  ip_set_descriptors {
    type  = "IPV4"
    value = "10.16.16.0/16"
  }
}
```

### Debug Output
N/A

### Panic Output
N/A 

### Expected Behavior
I would like to be able to import a waf_ipset. We created one manually in haste and now we'd like to properly manage it with terraform.

### Actual Behavior
```
aws_waf_ipset.dgn: Importing from ID "a1b2c3d4-d5f6-7777-8888-9999aaaabbbbcccc"...

terraform import aws_waf_ipset.ok_cidrs a1b2c3d4-d5f6-7777-8888-9999aaaabbbbcccc
Error: aws_waf_ipset.dgn (import id: a1b2c3d4-d5f6-7777-8888-9999aaaabbbbcccc): import aws_waf_ipset.ok_cidrs (id: a1b2c3d4-d5f6-7777-8888-9999aaaabbbbcccc): resource aws_waf_ipset doesn't support import
```

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform import aws_waf_ipset.main <id>`

### Important Factoids
N/A

### References
There are probably more WAF resources that don't support import; this is just the first one I ran into.

**Comment by bflad** [comment]
Pull request submitted: #4979

**Comment by bflad** [comment]
Resource import support for `aws_waf_ipset` has been merged in and will release with version 1.25.0 of the AWS provider, likely later this week.

**Comment by bflad** [comment]
This has been released in [version 1.25.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1250-june-27-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
